dojo.py

```diff
@@ -0,0 +1,37 @@
+from room import Office, Living_space
+from person import Fellow, Staff
+
+class Dojo(object):
```

Comment:
just a quick note -> I did some research yesterday and I found out that in python 3 you don't have to inherit 'object' explicitly. it is inherited implicitly by default. so '**_class Dojo(object):_**' is the same as just saying '**_class Dojo:_**'
[reference](http://stackoverflow.com/questions/4015417/python-class-inherits-object)